src/assets/user-info-settings/index.css

```
@@ -0,0 +1,298 @@
+html {
+	background: #FF0000;
+}
+
+body {
+	font-family: Poppins, Arial, Helvetica, sans-serif;
+	font-size: 15px;
+	width: 1280px;
+	margin: 0;
+	overflow: auto;
+	box-sizing: border-box;
+	background: #EAEAEA;
+}
+
+button:active,
+.server-selection input:checked + label {
+	box-shadow: inset 0 0px 10px 0 rgba(66, 45, 120, 0.75) !important;
+	color: #9D6FF3;
+}
+
+header {
+	position: relative;
+	background: #37A985;
+	color: #FFF;
+	height: 70px;
+	line-height: 70px;
+	border-bottom: 2px solid #4F2E8C;
+	background: -webkit-gradient(linear, left top, left bottom, from(#9D6FF3), to(#673DB6));
+	z-index: 19;
+}
+
+header h1 {
+	margin: 0;
+	font-size: 30px;
+	text-align: center;
+	font-weight: normal;
+}
+
+button,
+.button,
+input[type="submit"] {
+	font-family: Poppins, Arial, Helvetica, sans-serif;
+	color: #45297A;
+	min-width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+	min-height: 60px;
+	text-align: center;
+	font-size: 28px;
+	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB))0 0;
```

Nitpick. Missing space

```suggestion
	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB)) 0 0;
```